modules/core/src/test/scala/gem/LocationSpec.scala

```diff
@@ -70,7 +70,7 @@ class LocationSpec extends FlatSpec with Matchers with PropertyChecks with Arbit
     }
   }
 
-  it should "produce a sorted list of Location.Middle" in {
+  ignore should "produce a sorted list of Location.Middle" in {
```

Comment:
The recursive function in `find` is `@tailrec` but I take it that doesn't help JavaScript?